.clang-format

```diff
@@ -9,11 +9,28 @@ BinPackParameters: false
 BreakBeforeBinaryOperators: All
 BreakConstructorInitializers: BeforeColon
 ColumnLimit: 99
```

Comment:
i would add some comment pragmas for clang-tidy nolints

Reply:
Do you have a pointer handy to some time-tested config? Or a concrete suggestion? Happy to fold that in. (I just don't want to invent an incompatible wheel, not currently needing anything in particular.)

Reply:
[CommentPragmas](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) is a regular expression, can you just match it with a nolint comment? That's quite nice, actually.